std/format.d

```diff
@@ -569,66 +569,70 @@ matching failure happens.
 
 Throws:
     An `Exception` if `S.length == 0` and `fmt` has format specifiers
+
+Deprecated:
+
+The deprecation targets parameters of the form `&param` which take the address of
+a local and make most usages of this function non-@safe. Replace these parameters
```

Comment:
The deprecation (noun sg.) targets (verb in 3rd person) parameters (noun pl.) ...
I haven't had problems until now, but you're right. That's hard to read. I'll reformulate this.